pkg/scheduler/apis/config/v1beta2/default_plugins.go

```diff
@@ -116,16 +116,6 @@ func applyFeatureGates(config *v1beta2.Plugins) {
 	if utilfeature.DefaultFeatureGate.Enabled(features.VolumeCapacityPriority) {
 		config.Score.Enabled = append(config.Score.Enabled, v1beta2.Plugin{Name: names.VolumeBinding, Weight: pointer.Int32Ptr(1)})
 	}
-
```

Comment:
Since `PodTopologySpread` graduated to GA, should we remove `SelectorSpread` then? @alculquicondor